pkg/oc/generate/app/cmd/newapp.go

```diff
@@ -312,6 +313,7 @@ func (c *AppConfig) tryToAddTemplateArguments(s string) bool {
 // AddArguments converts command line arguments into the appropriate bucket based on what they look like
 func (c *AppConfig) AddArguments(args []string) []string {
 	unknown := []string{}
+	winnerFmt := "The argument %s passed the %s classification tests"
```

Comment:
can an argument be classified as more than one thing?  if not, i'd change this to "Argument x was classified as y"